app/pages/admin/project-status.jsx

```diff
@@ -124,17 +124,23 @@ class ProjectStatus extends Component {
       .catch(error => this.setState({ error }));
   }
 
+  setFeatured(project, value) {
+    project.update({ featured: value.checked });
+    return project.save()
+      .catch(error => this.setState({ error }));
+  }
+
   handleFeaturedProjectChange({ target }) {
     const { featured, project } = this.state;
     if (featured) {
```

Comment:
This is going to be false while the initial API call is waiting to resolve, so while you don’t know yet if there’s a featured project.